src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,360 @@
+           --data '{"parent_id":"<parent_id>"}'
+    ~~~
+
+    Replace:
+    - `<cluster-id>`: The ID of a cluster.
+    - `<api-key>`: Your service account's API key.
+    - `<parent-id>`: The ID of a folder.
+
+1. To move a cluster out of a folder to the organization level:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request PATCH \
+           --url 'https://management-staging.crdb.io/api/v1/clusters/<cluster-id>' \
+           --header 'Authorization: Bearer <api-key>' \
+           --header 'content-type: application/json' \
+           --data '{"parent_id":"root"}'
+    ~~~
+
+    Replace:
+    - `<cluster-id>`: The ID of a cluster.
+    - `<api-key>`: Your service account's API key.
+
+{{site.data.alerts.callout_info}}
+When you move a cluster into or out of a folder, users or service accounts who had access to the previous location through inheritance may lose access. Roles granted directly on a cluster do not change when the cluster is moved.
+{{site.data.alerts.end}}
+
+## Move a folder into another folder
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover)
+
+{{site.data.alerts.callout_info}}
+When you move a folder, users or service accounts who had access to the previous location through inheritance may lose access to its descendent folders and clusters. Roles granted directly on a folder or a cluster do not change when the folder is moved.
+{{site.data.alerts.end}}
+
+1. To move a folder and its contents into another folder:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request PATCH \
+           --url 'https://management-staging.crdb.io/api/v1/folders/<folder-id>' \
+           --header 'Authorization: Bearer <api-key>' \
+           --header 'content-type: application/json' \
+           --data '{"parent_id":"<parent_id>"}'
+    ~~~
+
+    Replace:
+    - `<folder-id>`: The ID of the folder.
+    - `<api-key>`: Your service account's API key.
+    - `<parent-id>`: The ID of the new parent folder.
+
+## Delete a folder
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin)
+
+{{site.data.alerts.callout_info}}
+Only an empty folder can be deleted.
+{{site.data.alerts.end}}
+
+To delete a folder:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request DELETE \
+       --url 'https://management-staging.crdb.io/api/v1/folders/<folder-id>' \
+       --header 'Authorization: Bearer <api-key>' \
+       --header 'content-type: application/json'
+~~~
+
+Replace:
+
+- `<folder-id>`: The ID of the folder.
+- `<api-key>`: Your service account's API key.
+
+## Limitations
+
+- Folders can be nested a maximum of three levels deep, including the organization level.
```

Comment:
@mdlinville same as above, not including the org level. Org > Folder 1 > Folder 2 > Folder 3 is how deep they can go.

Reply:
Done